### PR TITLE
fix(demo-pages accessibility): add missing labels, values, etc

### PR DESCRIPTION
**Pull request by julio8a**
**Related Issue:** #1576 

## Summary

Went through the list item on #1576 to make sure they have what's needed for our accessibility audit.

### CODE REVIEW

**Review by jcfranco — APPROVED** [review]
Awesome, Julio! 🎉 

Regarding the commit type, I think this would be either a `chore` or `test` type since this doesn't necessarily patch anything on the user's side:

```
chore(a11y): add missing labels, values, etc to demo pages
```

You can rewrite the commit type right before squash merging.

File: src/demos/action-bar/basic.html [hunk 1]
```diff
@@ -81,7 +81,7 @@ <h3>Tooltip on expand toggle</h3>
                   <calcite-action-bar id="action-bar-test">
                     <calcite-action text="Add" icon="plus"></calcite-action>
                   </calcite-action-bar>
-                  <calcite-tooltip id="tooltip">Hello world!</calcite-tooltip>
+                  <calcite-tooltip label="tooltip" reference-element="tooltip" id="tooltip">Hello world!</calcite-tooltip>
```
Comment (driskull) [inline src/demos/action-bar/basic.html:84]: @julio8a I don't think the reference element should be the same as the tooltip. The reference-element should be what triggers the tooltip. Applies to other places.

**Review by driskull — CHANGES_REQUESTED** [review]
Looks good just needs reference element changed

**Review by paulcpederson — APPROVED** [review]

**Comment by paulcpederson** [comment]
Yeah, I'd use `chore:` for this as well, otherwise looks great! Thanks @julio8a

File: src/demos/action-bar/basic.html (hunk 1 above)
Comment (julio8a) [reply src/demos/action-bar/basic.html]: Thanks! I've updated this.

**Pull request merged**